lib/css/CssGenerator.js

```diff
@@ -113,14 +137,36 @@ class CssGenerator extends Generator {
 		const data = generateContext.getData();
 		data.set("css-exports", cssExportsData);
 
-		return InitFragment.addToSource(source, initFragments, generateContext);
+		const generationResult = InitFragment.addToSource(
+			source,
+			initFragments,
+			generateContext
+		);
+		if (
+			module instanceof CssModule &&
+			generateContext.type === "javascript" &&
+			module.assertions &&
+			module.assertions.type === "css"
+		) {
+			generateContext.runtimeRequirements.add(RuntimeGlobals.module);
+			return new RawSource(`const __WEBPACK_CSS_STYLE_SHEET__ =  new CSSStyleSheet();
+__WEBPACK_CSS_STYLE_SHEET__.replaceSync(${convertToTemplateLiteral(generationResult.source().toString())});${RuntimeGlobals.module}.exports = __WEBPACK_CSS_STYLE_SHEET__;`);
```

Comment:
We should support generation without template literal too, we should use it only when [`output.environment.templateLiteral`](https://webpack.js.org/configuration/output/#outputenvironment) is `true`